python/openassetio/hostAPI/Manager.py

```diff
@@ -410,8 +409,7 @@ def isEntityReference(self, tokens, context):
         @see resolveEntityReference
 
         @todo Make use of
-        openassetio.constants.kField_EntityReferenceMatchPrefix or
-        openassetio.constants.kField_EntityReferenceMatchRegex if
+        openassetio.constants.kField_EntityReferenceMatchPrefix if
```

Comment:
There's an existing typo here, should be Reference**s**.

Reply:
Good spot. Fixed that [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/143/commits/301e748e609fbbff62f99067362466bf4cb75fc0).

Reply:
I noticed it's also wrong in Examples.dox.

Reply:
In fact. that needs updating anyway to talk about what's now the specific key, rather than 'keys'.

Reply:
That would explain way I missed it before when grepping for the key name with a `s`... Fixed [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/143/commits/f967ea46458bfb0938aed856c9d4a8e06b8657f2).